delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/FullyQualifiedImport.html

```diff
@@ -0,0 +1,32 @@
+<h2>Why is this an issue?</h2>
+<p>
+    Imports should be done using the full name of the unit, as this facilitates code readability, avoids potential ambiguities about which unit is being imported, and ensures consistency throughout the codebase.
+</p>
+<p>
+    Using the full name of the units not only improves code clarity and maintenance but also optimizes the compilation process. The compiler does not need to spend time searching implicit scopes to resolve which units are being referenced. By specifying the full unit name, the compiler can immediately locate the necessary files, resulting in faster and more efficient compilation.
+</p>
+<h2>How to fix it</h2>
+<p>
+    Fix the import to use the full name of the unit.
+</p>
+<h3>Non-compliant Code Example</h3>
+<pre data-diff-id="1" data-diff-type="noncompliant">
+unit VariableUtilsU;
+
+interface
+
+uses
+  Classes,
+  SysUtils;
```

Comment:
There's no need to include the top of the unit, especially with this arbitrary unit name - we can probably just show the uses clause. 

```suggestion
uses
  Classes,
  SysUtils;
```

Reply:
Done!